### DISCUSSION TITLE
The New Github Education Member.

### DISCUSSION THREAD

**Discussion by lzhbhlrPython** [community/community#40547] [GitHub Education]
Hello Everyone.
My name is Baolin Liu. I come from Tianjin China. I am a high school student of Tianjin No:1 High School of Binhai!

**Comment by jfullstackdev** [comment]
you can also introduce yourself at the official thread here,

https://github.com/orgs/community/discussions/23204

**Comment by samus-aran** [comment]
:wave: Welcome to the GitHub Community on Discussions, @lzhbhlrPython  ! Happy to have you here :slightly_smiling_face: 

It's been a while since you've created this discussion, so why don't you re-introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204). To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912).
